pagopa/io-app
# fix: [#175893821] Fixed status bar color inconsistency by making AppHeader a HOC

## Changes proposed in this pull request
- Status bar colors are now consistent when navigating through the app.
- Status bar is white/black text by default, to override this you need to edit the config variable in `AppHeader.tsx` specifying the route name and the color scheme.

## How to test 
Open the app and see the status bar's color consistent with the screen design: [Click to see video](https://user-images.githubusercontent.com/26572302/108636011-eb77ce80-7482-11eb-8e7e-01f5f033d1eb.mp4)

**Some inconsistencies remain in these screens (Android):**
<img width="309" height="auto" alt="checkmark-complete-status-bar-inconsistent" src="https://user-images.githubusercontent.com/26572302/108635997-d8fd9500-7482-11eb-9916-6fdeb8d28fe1.jpeg">

<img alt="loading-complete-status-bar-inconsistent" src="https://user-images.githubusercontent.com/26572302/108636002-ddc24900-7482-11eb-8959-93ef6534f781.png">

**And the splash screen (iOS).**
